Header header
uint32 poinsize
geometry_msgs/Pose[] poses
float32[] is_qualified
float32[] is_bundled
float64 direction_angle